src/identity/email-password-identity-provider/default-views/login.ejs

```diff
@@ -0,0 +1,20 @@
+<% if (locals.google) { %>
```

Comment:
Will there be a Google sign-in too? Should that be a separate thing?

Reply:
That's left over from something different. It will be removed. The main point of this was to get something that works as quickly as I could.